pkg/awsutils/awsutils.go

```diff
@@ -866,6 +866,11 @@ func (cache *EC2InstanceMetadataCache) AllocIPAddresses(eniID string, numIPs int
 		needIPs = ipLimit
```

Comment:
Should we log to let it be known that the ipLimit is less than the needed #?

Reply:
Right now we log that it the `tryAssignIPs()`